### ISSUE TITLE
Eldev fails to bootstrap on Emacs 26

### ISSUE THREAD

**Issue by ikappaki** [doublep/eldev#72]
It throws a `Trailing garbage following expression` error.

1. Download and locally extract Emacs 26 from http://ftp.gnu.org/gnu/emacs/windows/emacs-26/.
2. Open a Command prompt
3.  cd to eldev src
4. Point to 26.3 release and invoke eldev:
```batch
D:\src\eldev>set ELDEV_LOCAL=.

D:\src\eldev>set ELDEV_EMACS=d:/emacs-26.3-x86_64/bin/emacs.exe

D:\src\eldev>bin\eldev.bat
Trailing garbage following expression:
```
Although Emacs 26 is an old version, I believe it is good to have support for those Eldev users that have CI workflows that support older versions.

PR to follow.

**Comment by doublep** [comment]
Should be fixed by PR #73, right?

**Comment by ikappaki** [comment]
> Should be fixed by PR #73, right?

Correct, as implicitly referenced in this issue.
